sites/breadcrumbs/reports/pre-a11y-1.json

```
@@ -0,0 +1,621 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+  "@type": [
+    "earl:Assertion",
+    "acr:AssessmentReport"
+  ],
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/sites/breadcrumbs/breadcrumbs-en.html",
+    "dct:hasPart": [
+      {
+        "@id": "_:non-text-content_itm1",
+        "dct:isPartOf": "_:subject",
+        "@type": [
+          "earl:TestSubject",
+          "schema:WebPageElement"
+        ],
+        "dct:description": "Textual identification of the web page element that is being evaluated, like: 'Figure 2, a bar chart'. Provide a pointer when applicable (earl:pointer)",
+        "earl:pointer": {
+          "@type": "oa:CssSelector",
+          "@value": "img"
+        }
+      },
+      {
+        "@id": "_:video_abc",
+        "dct:isPartOf": "_:subject",
+        "@type": [
+          "earl:TestSubject",
+          "schema:WebPageElement"
+        ],
+        "dct:description": "Textual identification of the web page element that is being evaluated, like: 'Video ABC (earl:pointer)"
+      }
+    ],
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Describe what is being assessed, like the full page or a component inside that page. Provide a pointer when applicable (earl:pointer)",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": "title, meta, #wb-bc, main"
+    }
+  },
+  "earl:assertedBy": {
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": [
+      "earl:Assertor",
+      "foaf:Organization"
+    ],
+    "earl:mainAssertor": {
+      "foaf:name": "Shiva Kayathi (Github: @shiva-sc)",
+      "foaf:homepage": "https://github.com/shiva-sc",
+      "@type": [
+        "earl:Assertor",
+        "foaf:Person"
+      ]
+    }
+  },
+  "dct:date": null,
+  "dct:description": "Analyzing and exploring the subject and produce a pre-evaluation of all WCAG 2.1 SC at level AA.",
+  "acr:involvesExpertise": [
+    "wbv:acr/topics#aria",
+    "Javascript"
+  ],
+  "dct:source": "act:rulesets/wcag2x/wcag21_all_levelAA.json",
+  "acr:standard": "https://www.w3.org/TR/WCAG21",
+  "acr:conformanceOption": "act:standard/profiles/wcag#levelAA",
+  "earl:result": [
+    {
+      "earl:test": "WCAG21:non-text-content",
+      "earl:outcome": "earl:untested",
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
```

```suggestion
      "earl:mode": "earl:manual",
```
Please change earl:unknownMode to earl:manual. We do manual testing.